### DISCUSSION TITLE
Decap 3.1.0-beta.0

### DISCUSSION THREAD

**Discussion by martinjagodic** [decaporg/decap-cms#6954] [Announcements]
The main feature of this release is the upgrade to **React 18**. This is a large change with potential side effects, so we decided to release a beta version.

We kindly invite everyone to test, especially Gatsby users, who were most eagerly awaiting this update. If you find any new issues, please report them as [GitHub Issues](https://github.com/decaporg/decap-cms/issues).

NPM: https://www.npmjs.com/package/decap-cms/v/3.1.0-beta.0

GitHub Pull Request: https://github.com/decaporg/decap-cms/pull/6941

![decap-3 1 0-beta 0](https://github.com/decaporg/decap-cms/assets/42938138/653890f8-cf87-49eb-a402-1550ad521c45)

**Comment by Isaac-Tait** [comment]
This is great news! I am working on installing it now. First question is that the [docs](https://decapcms.org/docs/gatsby/#add-decap-cms-to-your-site) state we need to install `gatsby-plugin-netlify-cms` Is that true or is that dependency deprecated?

**Reply by martinjagodic** [reply]
Good question. I don't use Gatsby so I don't really know. I know there are efforts on Gatsby's side to update the plugin, but it's not published yet. I am working on setting up a Gatsby + Decap test project, I can get back to you when I will have that. But there's a good chance that you will figure out what to do before me.

**Reply by Isaac-Tait** [reply]
EDIT: This pull request seems to be getting more traction https://github.com/gatsbyjs/gatsby/pull/38554

It appears that there is a pull request to update the `gatsby-plugin-netlify-cms` plugin  https://github.com/gatsbyjs/gatsby/issues/38553

The name may change too to `gatsby-plugin-decap-cms`...

**Comment by Schotsl** [comment]
Great work!! 🥳
